katas/langtonant/micaela/functions.tests.js

```diff
@@ -0,0 +1,37 @@
+import { FuncMatrix, Movements } from './functions';
```

Comment:
Looks like the tests are not being run because file name contains `tests` instead of `test`, please fix the file name so the tests can be run